tests/suites/test_suite_ssl.function

```diff
@@ -1310,7 +1463,8 @@ void ssl_message_queue_overflow_underflow( )
     TEST_ASSERT( mbedtls_test_message_queue_pop_info( &queue, 1,
                                                       &test_message_len )
                  == MBEDTLS_TEST_ERROR_QUEUE_EMPTY );
-    TEST_ASSERT( test_message_len == 2 ); // Test that it remains unchanged
+    /* Test that the length remained unchanged */
+    TEST_ASSERT( test_message_len == 2 );
```

Comment:
This should be added in "Add a message metadata queue in ssl tests", not "Add a message-based socket mock connection to the ssl tests"